tests/test_compare.py

```diff
@@ -13,13 +14,33 @@
 
 @pytest.fixture(name="config", scope="module")
 def _config() -> Config:
-    """ Fixture that returns a static Config object. """
+    """Fixture that returns a static Config object."""
     return Config(float_absolute_tolerance=ABS_TOL, float_relative_tolerance=REL_TOL)
 
 
 # ===== Unit Tests =========================================
 
 
+@pytest.mark.parametrize(
+    "value, expected, is_equal",
+    [
+        (Unit("meter"), Unit("meter"), True),
+        (Unit("meter"), Unit("feet"), False),
+        (Unit("meter"), "meter", True),
+        ("meter", Unit("meter"), True),
+    ],
+)
+def test_compare_units(value, expected, config, is_equal):
+    """Test that units are compared as expected."""
+    # Arrange
```

Comment:
Plop this comment w the # Act comment below

Reply:
fixed [d2a3dfc](https://github.com/MORSECorp/snappiershot/pull/96/commits/d2a3dfc12a96bd7b9496c97b4b8a388d6995765e)